www-servers/sniproxy/sniproxy-0.5.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/dlundquist/sniproxy.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/dlundquist/sniproxy/archive/${PV}.tar.gz -> sniproxy-${PV}.tar.gz"
```

Comment:
You can use `${P}.tar.gz` on RHS of `->`, that value is custom Gentoo name, and it doesn't serve any specific purpose where extra verbosity would be useful.